### ISSUE TITLE
include deck.js in main hade index file

### ISSUE THREAD

**Issue by rangapv** [calvinmetcalf/deckdemo#1]
Hi,
I have downloaded deck.js, now in order to embed it in my main index.pug file so that the slide deck is in the main index file, what needs to be done  ? should i just do an include the deck.js directory ? 
Regards
Ranga

**Comment by calvinmetcalf** [comment]
I have no idea, I haven't touched this in like 7 years

**Comment by rangapv** [comment]
Ok, So basically what I am asking is ,  is this only a stand alone entity or does this also go into other pages like sticking at the middle of an another index page...if you can recollect, pls take ur time...

**Comment by calvinmetcalf** [comment]
i don't think this even works anymore, if you want to make a slide deck, this is almost certainly not the tree you want to be barking up, maybe try [reveal](https://revealjs.com/#/).

 I think this had to do with an old static site generator called docpad, which I'm not sure even exists.

**Comment by rangapv** [comment]
Ok, I will try revel.js instead, thanks!
